web/src/drafts_overlay_ui.ts

```diff
@@ -3,6 +3,7 @@ import _ from "lodash";
 import assert from "minimalistic-assert";
 
 import render_draft_table_body from "../templates/draft_table_body.hbs";
+import render_drafts_list from "../templates/drafts_list.hbs";
```

Comment:
Can you split this middle refactoring commit into several more fine-grained refactoring commits that are easy really easy to verify? That'd make this PR far easier to review and integrate.

Check out the [Zulip commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for more details on what I hav in mind.

Reply:
It looks like in this commit this file doesn't use `render_drafts_list`?